src/features/customerMessageForm/CustomerMessageForm.tsx

```diff
@@ -58,40 +58,70 @@ export const CustomerMessageForm = ({
       {({ values, errors, handleChange, handleSubmit }) => {
         return (
           <form onSubmit={handleSubmit} className={styles.form}>
-            <Text as="h4">{t('customerList.message.header')}</Text>
-            <Select
-              id="templateId"
-              value={values.templateId}
-              options={templateOptions}
-              onChange={handleChange}
-              label={t('customerList.message.template')}
-              required
-            />
-            <TextInput
-              id="subject"
-              value={values.subject}
-              onChange={handleChange}
-              label={t('customerList.message.subject')}
-              invalid={!!errors.subject}
-              helperText={errors.subject}
-            />
-            <TextArea
-              id="message"
-              className={styles.message}
-              value={values.message}
-              onChange={handleChange}
-              label={t('customerList.message.message')}
-              invalid={!!errors.message}
-              helperText={errors.message || t('customerList.message.messageHelperText')}
-            />
+            {!previewHtml ? (
+              <>
+                <Grid colsCount={2}>
+                  <Select
+                    id="templateId"
+                    value={values.templateId}
+                    options={templateOptions}
+                    onChange={handleChange}
+                    label={t('customerList.message.template')}
+                    required
+                  />
+                  <div className={styles.recipientCount}>
+                    {t('customerList.message.recipient', { count: recipientCount })}
+                  </div>
+                </Grid>
+                <TextInput
+                  id="subject"
+                  value={values.subject}
+                  onChange={handleChange}
+                  label={t('customerList.message.subject')}
+                  invalid={!!errors.subject}
+                  helperText={errors.subject}
+                  required
+                />
+                <TextArea
+                  id="message"
+                  className={styles.message}
+                  value={values.message}
+                  onChange={handleChange}
+                  label={t('customerList.message.message')}
+                  invalid={!!errors.message}
+                  helperText={errors.message || t('customerList.message.messageHelperText')}
+                />
+              </>
+            ) : (
+              <>
+                <p>
+                  {values.subject}　{t('customerList.message.recipient', { count: recipientCount })}
```

Comment:
There seems to be an alignment issue here, probably because of the whitespace is used in this markup.
<img width="260" alt="Screenshot 2020-08-19 at 16 38 50" src="https://user-images.githubusercontent.com/1407958/90642107-a6a47e00-e23a-11ea-88ec-d2270b560e49.png">

Reply:
Fixed 👍